net/freeradius3/Makefile

```diff
@@ -672,11 +712,8 @@ CONFIGURE_ARGS+= \
 	--with-radacctdir=/var/db/radacct \
 	--with-logdir=/var/log \
 	--without-pcre \
-	--without-rlm_cache \
 	--without-rlm_cache_memcached \
-	--without-rlm_couchbase \
```

Comment:
This is used by which module?

Reply:
openwrt doesn't have a couchbase package so I added it back ```--without-rlm_couchbase```